### ISSUE TITLE
[Enhancement] Design UI to match EMS records to crashes

### ISSUE THREAD

**Issue by patrickm02L** [cityofaustin/atd-data-tech#14654]
## Description
Companion Issue to #14592

As part of the MVP for connecting EMS incidents to crashes and persons, we need to provide a way to edit the linkage fields that will be added to the ems__incidents table (see related issue).

Discussion from 11/8/23 Dev Refinement

- Add an "Edit EMS" button on the People section in the crash record
- Create a separate Tab for EMS data that can be edited independently of the crash record

## Acceptance Criteria

- Design mocks of where/how EMS editing will occur
- Implement edit button connected to ems__incidents table
- Update person/crash pages to surface associated EMS data

## Notes
- @patrickm02L follow up with @Xavierapostol to get clarity on where to locate EMS editing capabilites to create design mock ups.

**Comment by chiaberry** [comment]
I see that Xavier also requested https://github.com/cityofaustin/atd-data-tech/issues/13713, to be able to edit the afd_incidents table as well, this design decision feels like it would influence that one and vice versa

**Comment by xavierapostol** [comment]
> I see that Xavier also requested #13713, to be able to edit the afd_incidents table as well, this design decision feels like it would influence that one and vice versa

That is correct. At minimum, I'd like to have the ability to associate an AFD Incident to a `crash_id`. 
**Note**: The AFD Incidents already have an associated EMS Incident ID in the field called `EMS_IncidentNumber`. So, those tables can be linked.

**Comment by patrickm02L** [comment]
## 11/20
Chatted with Xavier about where he preferred the EMS field be added. He asked for it to be added to the People's card, and to remove the City and Zip fields. He noted that nobody uses them and that Other people don't include them.

[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBM3p3QVE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--5e7756ec602c79cad93b7324e1341da57e2d2d2f/Screen%20Shot%202023-11-20%20at%201.45.19%20PM.png)

## Notes
- We discussed another tab for **Other Agencies** card which would house EMS, AFD, TCME data.
  - EMS would include `ems_incident_id`, `unit_nbr`, `prsn_nbr`
  - AFD would include `afd_incident_id`
  - TCME would include `tcme_case_id`, `unit_nbr`, `prsn_nbr`
- MVP - just EMS data in the Other Agencies card

**Comment by patrickm02L** [comment]
## 11/22
@patrickm02L to mock up the UI for other agencies card
- Think about how to represent the information for persons versus crashes
